### ISSUE TITLE
Logs filled with ingress api version deprecation warnings

### ISSUE THREAD

**Issue by alekitto** [containous/traefik#6536]
# Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?

Executing traefik 2.2.0-rc4 on kubernetes 1.17.2, logs are filled with

```
{"level":"warning","msg":"Ingress ns/ingress: the apiVersion 'extensions/v1beta1' is deprecated, use 'networking.k8s.io/v1beta1' instead.", "time": "..."}
```

This warning is printed out at least once per second.
I've updated all the ingresses to `networking.k8s.io/v1beta1` api version, but nothing has changed.

### What did you expect to see?

No warnings.

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      2.2.0-rc4
Codename:     chevrotin
Go version:   go1.14
Built:        2020-03-19T17:31:45Z
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

Running on kubernetes 1.17.2

**Comment by jbdoumenjou** [comment]
Hi @overheadhunter 
I'm sorry, I fixed it on master instead of v2.2, it will come in the coming days.

**Comment by traefiker** [comment]
Closed by #6624.
